casper/src/main/resources/PoS.rhox

```diff
@@ -299,11 +300,11 @@ new  PoS,
                   new toBeSlashed in {
                     toBeSlashed!(invalidBlocks.getOrElse(blockHash, userPk)) |
                     for (@validator <- toBeSlashed) {
+                      // TODO: remove pending rewards
```

Comment:
This is still unfinished. @pyrocto I think is enough to map over all validators pending reward maps and collect (and set to zero) current values.

Reply:
It's finished now.

Reply:
It's finished now.

Reply:
Here is the ticket to write tests for slashing.
https://rchain.atlassian.net/browse/RCHAIN-3984